### DISCUSSION TITLE
Is there any way to run mirror.exe in the background?

### DISCUSSION THREAD

**Discussion by ArduinoBen** [dokan-dev/dokany#1117] [Q&A]
I am currently running `mirror.exe -o -r \\NAS\SHARE -l M` through cmd, to mount an SMB share as a removable drive.  I am able to minimize the window but not close it out.

Thus, I was wondering if there was a way to run mirror.exe in the background, so that the program wouldn't show up in my taskbar.  I'm assuming it's possible but likely somewhat complicated.  Still, I thought I'd ask in case there was an easy way to do this.

I'm running dokany on Windows 11 if that makes any difference.

Thanks.

**Comment by Liryna** [comment]
Hi @ArduinoBen , you might find the answer in this thread https://github.com/dokan-dev/dokany/discussions/1005
